src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -339,14 +402,14 @@ private StatusResponsePayload ConvertFrom(DurableOrchestrationStatus status)
             };
         }
 
-        private static IEnumerable<OrchestrationRuntimeStatus> GetIEnumerableQueryParameterValue(NameValueCollection queryStringNameValueCollection, string queryParameterName)
+        private static IEnumerable<T> GetIEnumerableQueryParameterValue<T>(NameValueCollection queryStringNameValueCollection, string queryParameterName) where T : struct

```

Comment:
Can you explain this change to me?

Reply:
Hello @cgillum,

Thank you for the comment. I wanted to reuse this method originally created by Tsuyoshi instead of implementing new one that will be actually the same. The originally had hard-coded return type of `IEnumerable<OrchestrationRuntimeStatus>` and I wanted to use it for `IEnumerable<OrchestrationStatus>` so I decided to parametrize it and reuse it. Let me know if you prefer to go with new dedicated method.

Thank you!